model.py

```diff
@@ -150,6 +150,8 @@ def toggle(self, uid, user_profile):
 
     def render_text(self):
         title = util.make_html_bold(self.title)
+        if self.people:
+            title += ' ({} {})'.format(len(self.people), util.emoji_people_unicode())
```

Comment:
Just tested it on a staging server; the string needs to be unicode otherwise it'll result in an encoding error. Also, it looks slightly better without the space:

```python
title += u' ({}{})'.format...
```